P2PEnabledConference.js

```diff
@@ -0,0 +1,726 @@
+/* global __filename */
+
+import {getLogger} from "jitsi-meet-logger";
+const logger = getLogger(__filename);
```

Comment:
Yes, I agree. I did it only, because I saw it done like this in few other places already, but will change that.